numpy/f2py/f2py2e.py

```diff
@@ -167,12 +163,12 @@
   array. Integer <int> sets the threshold for array sizes when
   a message should be shown.
 
-Version:     %s
-numpy Version: %s
+Version:     {f2py_version}
+numpy Version: {numpy_version}
```

Comment:
Seems kinda pointless to show this twice

Reply:
Agree, but it might break something downstream if removed. My intent was to keep the changes isolated in the `__version__.py` file in case we later want to make other changes in reporting the version.

Hmm, the broken test is interesting and I'm not sure how to fix it so that is also runs with `runtests.py`, which seems to use the system installed `f2py` rather than the new build. Looks like the test has always been broken anyway as previously the f2py version never changed.